From 59193f066dbf4318f95d5f4740df1374565e09e1 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Tue, 18 Nov 2014 14:16:39 +0000
Subject: [PATCH] v2v: linux: Always match partition number in regexp.

Change the rex_device_p regular expression to always include a
partition number.

There should be no functional change here.

(cherry picked from commit b1053150a2da121b3cb8ddc197d0d8e49d930130)
---
 v2v/convert_linux.ml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/v2v/convert_linux.ml b/v2v/convert_linux.ml
index 2f97ca1..d094969 100644
--- a/v2v/convert_linux.ml
+++ b/v2v/convert_linux.ml
@@ -1300,7 +1300,7 @@ let rec convert ~verbose ~keep_serial_console (g : G.guestfs) inspect source =
     and rex_device_cciss =
       Str.regexp "^/dev/\\(cciss/c[0-9]+d[0-9]+\\)$"
     and rex_device_p =
-      Str.regexp "^/dev/\\([a-z]+\\)\\([0-9]*\\)$"
+      Str.regexp "^/dev/\\([a-z]+\\)\\([0-9]+\\)$"
     and rex_device =
       Str.regexp "^/dev/\\([a-z]+\\)$" in
 
-- 
1.8.3.1

